conference.js

```diff
@@ -2792,6 +2817,12 @@ export default {
                                     // Use the new stream or null if we failed to obtain it.
                                     return useStream(tracks.find(track => track.getType() === mediaType) || null)
                                         .then(() => {
+                                            if (hasDefaultMicChanged) {
+                                                // workaround for the default device to be shown as selected in the
+                                                // settings even when the real device id was passed to gUM because of
+                                                // the above mentioned chrome bug.
+                                                this.localAudio._realDeviceId = this.localAudio.deviceId = 'default';
```

Comment:
IMHO this seems a little bit dirty. Do you think we'll need this workaround permanently or you plan to remove it relatively soon?

If we plan to remove it relatively soon I guess it is fine!

Maybe another way to implement it is to store a flag which indicates if the default device is selected or not somewhere in the redux store and use it with priority when you show what is selected.

Reply:
This is just a workaround until the chrome bug is fixed, plus it’s only applicable to chrome since the other browsers don’t enumerate default devices separately.